libs/wire-subsystems/test/unit/Wire/MiniBackend.hs

```diff
@@ -72,6 +80,24 @@ import Wire.UserSubsystem
 import Wire.UserSubsystem.Error
 import Wire.UserSubsystem.Interpreter
 
+newtype PendingNotEmptyIdentityStoredUser = PendingNotEmptyIdentityStoredUser StoredUser
+  deriving (Show, Eq)
+
+instance Arbitrary PendingNotEmptyIdentityStoredUser where
+  arbitrary = do
+    user <- arbitrary `suchThat` \user -> isJust user.identity
```

Comment:
why not create a user identity first, then update the generated user with that identity? That way our tests don't slow down because of unnecessary discards. it also increases coverage.

Reply:
non-trivial (identity is not a field of stored user, but a function in several fields).

separate PR!